velox/exec/RowContainer.cpp

```diff
@@ -530,4 +522,272 @@ void RowContainer::setProbedFlag(char** rows, int32_t numRows) {
   }
 }
 
+RowTypePtr RowContainer::rowType() const {
+  VELOX_CHECK(aggregates_.empty(), "Aggregate type info not yet implemented");
+  auto copy = types_;
+  return ROW(std::move(copy));
+}
+
+void RowContainer::extractSpill(
```

Comment:
This method simply copies 'rows' into a vector. What does it have to do with spilling?